app/api-v1/api-doc.js

```diff
@@ -271,11 +304,10 @@ const apiDoc = {
           },
           items: {
             type: 'array',
-            description: 'List of parts to be supplied',
+            description: 'List of recipes to be supplied',
             maxItems: 10,
             items: {
-              description: 'id of the recipe to be built',
-              allOf: [{ $ref: '#/components/schemas/ObjectReference' }],
+              $ref: '#/components/schemas/ObjectReference',
```

Comment:
using just `ref` instead of `allOf` means it shows an example for `ObjectReference`, but we can't set `description` so maybe it would be better just to keep using `allOf` and add an `example` here. I was thinking the top level description `List of recipes to be supplied` might be enough

Reply:
reverted and added `examples` instead